### PR TITLE
Created Test Suite - IFTAQCAA-18

### PR DESCRIPTION
**Pull request by Ferrl-m**

### CODE REVIEW

File: testng.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name = "Suite1">
```
Comment (Nikodych) [inline testng.xml:4]: create another one for difference, with a list of test methods

<test name="">
        <classes>
            <class name="">
                <methods>
                    <include name=""/>
                    <include name=""/>
                </methods>
            </class>
        </classes>
    </test>

and add to any of the suites such config

<suite name="" parallel="tests" thread-count="1">

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# ifa-134.ta
\ No newline at end of file
+# ifa-134.ta
+How to run Test Suite
```
Comment (Nikodych) [inline README.md:2]: also, add instruction on how to run through the console with mvn command

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Comment by Ferrl-m** [comment]
using mvn clean test -DsuiteXmlFile=fileName.xml makoviichukTests works correctly 
image

but allTests with same tests fails 
image

File: allTests.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name = "All Tests">
```
Comment (Nikodych) [inline allTests.xml:4]: so you are able for all tests to use the previous approach, use this approach for such suite as "Smoke suite" and choose few test with this approach

File: pom.xml [hunk 4]
```diff
@@ -19,8 +19,26 @@
...
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <suiteXml>allTests.xml</suiteXml>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```
Comment (Nikodych) [inline pom.xml:30]: probably Pavlo also imported this plugin, and it's already merged

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Review by Nikodych — APPROVED** [review]

**Pull request merged**